whereby/dev-docs
# Update http-api docs to include streaming option

Document the new `streaming` option on meeting creation.

To test:
`yarn start` -> https://localhost:8080/http-api -> verify streaming option is documented
